@kiva/kv-tokens/primitives.js

```diff
@@ -400,16 +417,27 @@ export default {
 		4: 32,
 		4.5: 36,
 		5: 40,
+		5.5: 44,
```

Comment:
These `.5` variations are needed for these changeset? I'm not seeing them used in the templates.